docs/en/connector-v2/sink/Elasticsearch.md

```diff
@@ -50,8 +50,8 @@ x-pack password
 ### max_retry_size [int]
 one bulk request max try size
 
-### max_batch_size [int]
-batch bulk doc max size
```

Comment:
Considering that there is no official version released, this incompatible change only needs to be reflected in the `changelog` (Next version).

Reply:
OK